### ISSUE TITLE
[Question] How to configure continue training for a model 

### ISSUE THREAD

**Issue by Dhanasekar-S** [modelscope/AdaSeq#11]
### What is your question?

I trained a baseline checkpoint for 50 epochs with a LinearLR schedule starting from 1.0 to 0.0 in 50 iters. If I try to continue training that checkpoint on the same dataset or a new dataset, the learning rate is set to zero, eval accuracy remains the same even though the loss is decreasing. Can you please help me continue training so that I can use existing baselines on new datasets

### What have you tried?

I tried training a model with a baseline config and update it with a new config to continue training. 

### What's your environment?

- AdaSeq Version (e.g., 1.0 or master):
- ModelScope Version (e.g., 1.0 or master):
- PyTorch Version (e.g., 1.12.1):
- OS (e.g., Ubuntu 20.04):
- Python version: 3.7.5
- CUDA/cuDNN version:
- GPU models and configuration:
- Any other relevant information:

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by huangshenno1** [comment]
@Dhanasekar-S Sorry for the lack of documentation. We are now working hard to complete it.
You can add `load_all_state: false` in your config file, and then run `python scripts/train.py -c ${config_file} -ckpt ${checkpoint_path}` to do continue training.

An example snippet:
```
  hooks:
    - type: BestCkptSaverHook
      save_file_name: best_model.pth
      metric_key: macro-f1
      save_optimizer: false
      restore_best: true
      load_all_state: false
    - type: CheckpointHook
      load_all_state: false
```
